config/config.exs

```diff
@@ -2,29 +2,6 @@
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
 
-# This configuration is loaded before any dependency and is restricted
-# to this project. If another project depends on this project, this
-# file won't be loaded nor affect the parent project. For this reason,
-# if you want to provide default values for your application for
-# 3rd-party users, it should be done in your "mix.exs" file.
-
-# You can configure your application as:
-#
-#     config :atrium, key: :value
-#
-# and access this configuration in your application as:
-#
-#     Application.get_env(:atrium, :key)
-#
-# You can also configure a 3rd-party app:
-#
-#     config :logger, level: :info
-#
-
-# It is also possible to import configuration files, relative to this
-# directory. For example, you can emulate configuration per environment
-# by uncommenting the line below and defining dev.exs, test.exs and such.
-# Configuration from the imported file will override the ones defined
-# here (which is why it is important to import them last).
-#
-#     import_config "#{Mix.env}.exs"
+config :atrium, api_key: "TEST_API_KEY",
+                client_id: "TEST_CLIENT_ID",
+                environment: "http://localhost:5000"
```

Comment:
Is `environment` the typical verbiage used for this url in other atrium projects? At first glance, it just makes me think it does more than a url, but it doesn't. 

I'm good to 🚢 as is, but if we wanted to consider changing this to `host`, `base_url`, `atrium_url`, or something like that then I'd be 👌 with that.

Reply:
hmmm, good question. It looks like [atrium-ruby](https://github.com/mxenabled/atrium-ruby#atrium-ruby) calls it `base_url`